### ISSUE TITLE
Skrell skin-color bugs out with prosthetic limbs

### ISSUE THREAD

**Issue by Swim420** [Aurorastation/Aurora.3#266]
As a Skrell, when I have a prosthetic limb it makes my skin this weird aquamarine color (shown below), as compared to the pink-purple(hair color) that it should be.

![skin](https://cloud.githubusercontent.com/assets/18385787/14413750/32f404d6-ff49-11e5-92fe-f69228d8b631.PNG)

**Comment by Akrilla** [comment]
"There is a bug with /obj/item/organ/external/proc/get_icon(var/skeletal)
That causes these colour issues with IPC and skrell
Wherever this function is called, problems happen.
And funnily enough it's called in /obj/item/organ/external/robotize
Which is the issue with skrell (colour going odd with a robotic arm)
And it's also called when getting body colour as well
I.e. with IPC's problem
And most likely it's causing other issues we haven't noticed yet
For the sake of providing all the details I've found, the chest update is here /mob/living/carbon/human/proc/update_body"

Copy + paste from my rant about this issue in Discord. I'm a bit burnt out on it, so I'll come back to it later, but the above is what I found.
